src/content/troubleshooting/_index.en.md

```diff
@@ -0,0 +1,136 @@
+### Connectivity Issues
+Deployment went through successfully but services/pods on one cluster are unable to connect to services in another clusters. This can be due to multuple factors - IP Sec tunnels, IP Table rules, Gateway pods, Routeagent etc.
+
+#### IPSec tunnel not created between clusters
+TBD
+
+#### IPSEc tunnel is not up between clusters
+TBD
+
+#### None of pods/services able to connect to remote service
+TBD
+##### Without Globalnet
+TBD
+##### With Globalnet
+TBD
+
+#### Pods on non-gateway nodes not able to connect to remote service
+TBD
+##### Without Globalnet
+TBD
+##### With Globalnet
+TBD
+
+### Service Discovery Issues
+If you are able to connect to remote service by using ServiceIP or GlobalnetIP, but not by service name, it is a Service Discovery Issue.
+
+#### Service Discovery not working
+This is good time to familiarize yourself with [Lighthouse Architecture](../architecture/components/lighthouse) if you haven't already.
+
+##### Check CoreDNS Configuration
+Submariner configures CoreDNS deployment to enable `lighthouse` plugin. Since CoreDNS is the first point of entry for any DNS queries, that is where we will start first. 
+
+First we check if we are running the correct image. We need CoreDNS image with ligthouse enabled.
+
+```kubectl -n kube-system kube-dns describe deployment```
+
+Make sure that `Image` is set to `quay.io/submariner/lighthouse-coredns:<version>` and version is correct. If not, change it to use the Lighthouse image.
+
+If image is correct, next we need to check if `lighthouse` plugin is enabled on the CoreDNS in the source cluster i.e. the cluster making the query.
+
+```kubectl describe configmap coredns```
+
+In the output look for something like this:
+
+```
+        kubernetes cluster2.local in-addr.arpa ip6.arpa {
+           pods insecure
+           upstream
+           #fallthrough in-addr.arpa ip6.arpa
+           fallthrough     =====> This tells kubernetes to go to next plugin if query fails locally
+        }
+        lighthouse { ====> Lighthouse plugin will catch any queries that local kubernetes can't resolve
+           fallthrough =====> proceed as usual if lighthouse can't resolve
+        }
+```
+If the entries highlighted above are missing, it means CoreDNS wasn't configured with lighthouse. It can be easily added by running `kubectl edit configmap coredns` and making the changes manually. You may need to repeat this step on every cluster.
+
+##### Check submariner-lighthouse-agent
+Next we check if `submariner-lighthouse-agent` is running correctly or not. Run `kubectl -n submariner-operator get pods submariner-lighthouse-agent` and check status of pods.
+
+If status is error with `ImagePullBackOff`, run `kubectl -n submariner-operator describe deployment submariner-lighthouse-agent` and check if `Image` is set correctly to `quay.io/submariner/lighthouse-agent:<version>`. If it is and still getting the same error, raise an issue as image has likely gone missing from quay.io
```

Comment:
```suggestion
If status is error with `ImagePullBackOff`, run `kubectl -n submariner-operator describe deployment submariner-lighthouse-agent` and check if `Image` is set correctly to `quay.io/submariner/lighthouse-agent:<version>`. If it is and still getting the same error, raise an issue [here](https://github.com/submariner-io/lighthouse/issues) or ping us on the community slack channel.
```

Reply:
> 

Also add link to community slack channel

Reply:
@Jaanki Good idea, but think we need a "Contact" section on website and link to it.